### ISSUE TITLE
error installing, spinner

### ISSUE THREAD

**Issue by nikomatsakis** [brotzeit/rustic#65]
When I try to install I get this:

```
Error (use-package): spinner/:catch: Loading file /home/nmatsakis/.emacs.d/elpa/spinner-1.7.3/spinner.elc failed to provide feature ‘spinner’
Error (use-package): Cannot load spinner
```

I found this similar bug report https://github.com/clojure-emacs/cider/issues/1843 and *tried* deleting spinner from my M-x package-list and reinstalling, but I'm not able to install, as I just get

```
https://elpa.gnu.org/packages/spinner-1.7.3.el: Bad Request
```

I realize this is probably not a bug with rustic, but I'd be happy to get some advice nonetheless! 

Other context:

GNU Emacs 26.1 (build 1, x86_64-redhat-linux-gnu)
 of 2018-08-13

**Comment by nikomatsakis** [comment]
cc @brotzeit -- I mentioned errors trying to build :)

**Comment by brotzeit** [comment]
Thanks for the bug report. I can't even find spinner in melpa anymore.

**Comment by nikomatsakis** [comment]
I have no idea how spinner is related to rustic, also, is it a dependency of yours?

**Comment by brotzeit** [comment]
It's supposed to indicate a running process for org-babel. IMO it's better to use existing packages instead of reimplementing its functionality. I know rustic has too many dependencies and I intend to remove some of them, but spinner is fine. However this is obviously a melpa issue.

**Comment by nikomatsakis** [comment]
No judgement for using dependencies :) I'm a big fan. I was just curious.

**Comment by brotzeit** [comment]
Yeah, but there is at least one package that is only used for a function that can be easily replaced. Somebody complained about it some time ago ;)

**Comment by brotzeit** [comment]
I just realized spinner is only in the gnu archive...
I will open a PR on melpa. To install spinner you have to add gnu to your `package-archives`:

```elisp
(setq package-archives '(("melpa" . "http://melpa.org/packages/")
                                     ("gnu" . "http://elpa.gnu.org/packages/")))
```
I will also add this info to the readme.

**Comment by brotzeit** [comment]
Ah your error complains about bad request of gnu elpa. You really can't install spinner with gnu added to package archives ?

**Comment by brotzeit** [comment]
Hmm, spinner also make the tests fail.

**Comment by brotzeit** [comment]
Can you try it with updating the elpa keys #68

`gpg2 --keyserver hkp://pool.sks-keyservers.net:80 --homedir $HOME/.emacs.d/elpa/gnupg --recv-keys 066DAFCB81E42C40`

**Comment by brotzeit** [comment]
I documented it in the readme. You can find instructions here https://elpa.gnu.org/packages/gnu-elpa-keyring-update.html
